### PR TITLE
Add first test to the project

### PR DESCRIPTION
**Pull request by sectioneight**
Also enable coveralls.

References #6

### CODE REVIEW

**Comment by sectioneight** [comment]
@jjx

File: src/test/java/com/uber/jenkins/phabricator/conduit/ArcanistClientTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ArcanistClientTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    Map<String, String> emptyParams = new HashMap<String, String>();
```
Comment (jjx) [inline src/test/java/com/uber/jenkins/phabricator/conduit/ArcanistClientTest.java]: Let's make this private static final

**Comment by jjx** [comment]
lgtm!

File: src/test/java/com/uber/jenkins/phabricator/conduit/ArcanistClientTest.java (hunk 1 above)
Comment (sectioneight) [inline src/test/java/com/uber/jenkins/phabricator/conduit/ArcanistClientTest.java]: done

**Comment by sectioneight** [comment]
8.47% is pretty :shit: but at least better than 0 :\

**Pull request merged**